docker/parameters.yaml.docker

```diff
@@ -0,0 +1,29 @@
+# parameters.yml.dist is a template
```

Comment:
I don't think we need this comment. Otherwise change it so that uses the correct filename :)